projects/js-packages/charts/src/components/pie-semi-circle-chart/stories/index.stories.tsx

```diff
@@ -7,18 +7,24 @@ const data = [
 		value: 80000,
 		valueDisplay: '$80K',
 		percentage: 2,
+		color: '#3858E9',
+		index: 0,
```

Comment:
I don't think that index is being used. When accessing a Datum object it provides its own index that is generated by the library so I don't think we need to define it here.